### ISSUE TITLE
virtualbox: Building Ubuntu 14.04 hangs

### ISSUE THREAD

**Issue by mgruner** [chef/bento#1141]
# Version:

Bento as of latest git master (05772bd2eca115d5bb4b2968e9fa2701d7bce17b)

VirtualBox 5.2.22

# Scenario:

Build the ubuntu 14.04 box.

# Expected Result:

Rebuild completes successfully.

# Actual Result:

Rebuild fails after a timeout, trying to start the VirtualBox additions without success.

```
==> virtualbox-iso: Provisioning with shell script: ../_common/virtualbox.sh
    virtualbox-iso: + HOME_DIR=/home/vagrant
    virtualbox-iso: + cat /home/vagrant/.vbox_version
    virtualbox-iso: + VER=5.2.22
    virtualbox-iso: + ISO=VBoxGuestAdditions_5.2.22.iso
    virtualbox-iso: + mkdir -p /tmp/vbox
    virtualbox-iso: + mount -o loop /home/vagrant/VBoxGuestAdditions_5.2.22.iso /tmp/vbox
    virtualbox-iso: mount: block device /home/vagrant/VBoxGuestAdditions_5.2.22.iso is write-protected, mounting read-only
    virtualbox-iso: + sh /tmp/vbox/VBoxLinuxAdditions.run
    virtualbox-iso: Verifying archive integrity... All good.
    virtualbox-iso: Uncompressing VirtualBox 5.2.22 Guest Additions for Linux........
    virtualbox-iso: VirtualBox Guest Additions installer
    virtualbox-iso: Copying additional installer modules ...
    virtualbox-iso: Installing additional modules ...
    virtualbox-iso: VirtualBox Guest Additions: Building the VirtualBox Guest Additions kernel modules.  This may take a while.
    virtualbox-iso: VirtualBox Guest Additions: Starting.
==> virtualbox-iso: Deregistering and deleting VM...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Build was cancelled.
```

This has been happening for several patch level releases of VirtualBox for me.

**Comment by cheeseplus** [comment]
Confirming this behaviour for amd64 and i386 templates. I've not dug into VBox issues to see what might be causing it but we could also try switching to an HWE kernel to see if that works around the issue.

**Comment by tas50** [comment]
This has been fixed by updating the ISO to 14.04.4 from 14.04.1

**Comment by mgruner** [comment]
Confirmed. Thank you very much @tas50!
